config_skeleton.gemspec

```diff
@@ -16,9 +16,8 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.3.0"
 
   s.add_runtime_dependency 'diffy', '~> 3.0'
-  s.add_runtime_dependency 'frankenstein', '~> 1.0'
```

Comment:
🤔 `frankenstein` is still a direct dependency of this gem – why is this gone from the gemspec?

Reply:
🤦 fixed, thanks!

Reply:
Oh I remember why now. Frankenstein is a transitive dependency of service_skeleton. So there is no need to pin a specific version here (and, in fact, pinning 1.0 breaks things).

Reply:
It's not just transitive dependency, `config_skeleton` also uses it directly. It should be declared. (can be added without specifying version)